ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,28 @@
+_PANDAS_VERSION="0.19.1"
+_NUMEXPR_VERSION="2.6.0"
+_BOTTLENECK_VERSION="1.2.0"
+
+config()
+{
+	# Verify that a compatible module can be found exists
+	./scripts/check_python_module -v pandas $_PANDAS_VERSION | eups_console
+
+	if [[ ${PIPESTATUS[0]} -ne 0 ]]; then
+		die "Failed to find a compatible externally provided pandas."
+	fi
+
+        ./scripts/check_python_module -v numexpr $_NUMEXPR_VERSION | eups_console
```

Comment:
numexpr and bottleneck are needed by some, but not all, pandas functions.  Since the three are intimately connected, I figured I would just check for all three here, rather than add another two eups stubs to our list of repositories (also: because it is not transparent within pandas what requires numexpr and bottleneck, it will not be obvious to developers when they depend on numexpr and bottleneck; it seemed safer to demand that a dependence on pandas was a dependence on numexpr and bottleneck).